examples/rate_against_momentum.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.constants import Boltzmann
+from surface_potential_analysis.basis.time_basis_like import EvenlySpacedTimeBasis
+
+from coherent_rates.isf import get_ak_data_1d
+from coherent_rates.system import (
+    HYDROGEN_FREE,
+    HYDROGEN_NICKEL_SYSTEM,
+    PeriodicSystemConfig,
+)
+
+config = PeriodicSystemConfig((20,), (50,), 50, temperature=155)
+system = HYDROGEN_NICKEL_SYSTEM
+
+times = EvenlySpacedTimeBasis(100, 1, 0, 1e-13)
+
+(xdata, ydata) = get_ak_data_1d(system, config, times, n_points=8)
+plt.plot(xdata, ydata, "bo", label="HNi")
+m, b = np.polyfit(xdata, ydata, 1)
+plt.plot(xdata, m * xdata + b, "b")
+
+mass = Boltzmann * config.temperature / (m * m)
```

Comment:
write a function that does this and takes in a value list so it is clear what it is doing